lib/src/checkbox/Checkbox.stories.tsx

```diff
@@ -84,44 +68,16 @@ const Checkbox = () => (
           <DxcCheckbox label="Checkbox" defaultChecked />
         </ExampleContainer>
         <ExampleContainer>
-          <Title title="Optional" theme="dark" level={4} />
-          <DxcCheckbox label="Checkbox" optional />
+          <Title title="Disabled" theme="dark" level={4} />
+          <DxcCheckbox label="Checkbox" disabled />
         </ExampleContainer>
         <ExampleContainer>
           <Title title="Disabled and checked" theme="dark" level={4} />
           <DxcCheckbox label="Checkbox" disabled defaultChecked />
         </ExampleContainer>
-        <ExampleContainer>
-          <Title title="Disabled and optional" theme="dark" level={4} />
-          <DxcCheckbox label="Checkbox" disabled optional />
-        </ExampleContainer>
-        <ExampleContainer>
-          <Title title="Disabled, optional and checked" theme="dark" level={4} />
-          <DxcCheckbox label="Checkbox" disabled optional defaultChecked />
-        </ExampleContainer>
-        <ExampleContainer>
-          <Title title="Label after" theme="dark" level={4} />
-          <DxcCheckbox label="Checkbox" labelPosition="after" />
-        </ExampleContainer>
-        <ExampleContainer>
-          <Title title="Checked with label after" theme="dark" level={4} />
-          <DxcCheckbox label="Checkbox" defaultChecked labelPosition="after" />
-        </ExampleContainer>
-        <ExampleContainer>
-          <Title title="Optional with label after" theme="dark" level={4} />
-          <DxcCheckbox label="Checkbox" optional labelPosition="after" />
-        </ExampleContainer>
-        <ExampleContainer>
-          <Title title="Disabled and checked with label after" theme="dark" level={4} />
```

Comment:
I would not remove some scenarios like the disabled optional.

Reply:
Why should we test that? `(Optional)` text is just been added as plain text inside the `<Label>` component. It would be the same as testing the disabled with a different string each time.

Reply:
It's not about testing the string but making sure that even disabled, the `(Optional)` string is still added with the correct styles.

Reply:
The presence of the optional label doesn't depend on whether the component is disabled or not. And, as I said, is just a string inside the same component as the `label` prop, tested one, tested all. I can't see the value here Jiale.

Reply:
I still do not agree, we should not program for the future (in this kind of situation) since we do not know if the component will evolve and if a dependency will appear here. Anyways, I add the test to unlock this PR.